tests/strict-equality.test.js

```diff
@@ -1,70 +1,58 @@
 const describe = require('mocha').describe
 const it = require('mocha').it
 const expect = require('chai').expect
-const strictequal = require('../strict-equality')
+const strictEqual = require('../strict-equality')
 
 
 describe('strict-equality', () => {
   describe('compare', () => {
-    it('determines inputone and inputtwo as equal numbers', () => {
+    it('determines 2 and 2 are equal numbers', () => {
```

Comment:
Tests should focus on scenarios not values. Since strictEqual works by checking the type and value you'd want 4 tests

1) Same type, same value (ex. 10, 10)
2) Same type, different value (ex. 10, 15)
3) Different type, same value (ex. 10, '10')
4) Different type, different value (ex. 10 '15')

Because of this 10,10 and 'a','a' would be the same test since they are both same type, same value.

You got there with your tests but you want to make sure that your descriptions read that way

```JavaScript
it('returns true when the parameters are the same type and same value', () => {
```